dev/packages/alpha/cisco-0.0.2/dataset/asa/manifest.yml

```diff
@@ -0,0 +1,52 @@
+title: Cisco asa logs
+release: ga
+type: logs
+streams:
+  - input: logs
+    vars:
+      - name: paths
+        type: text
+        title: Paths
+        multi: true
+        required: true
+        show_user: true
+        default:
+          - /var/log/cisco-asa.log
+      - name: tags
+        type: text
+        title: Tags
+        multi: true
+        required: true
+        show_user: true
+        default:
+          - cisco-asa
+      - name: syslog_port
+        type: integer
+        title: Syslog Port
+        multi: false
+        required: true
+        show_user: true
+        default: 9001
+      - name: log_level
+        type: integer
+        title: Log Level
+        multi: false
+        required: true
+        show_user: true
+        default: 7
+      - name: internal_prefix
+        type: text
+        title: Internal Prefix
+        multi: false
+        required: true
+        show_user: true
+        default: asa
+      - name: internal_PREFIX
+        type: text
+        title: Internal PREFIX
+        multi: false
+        required: true
+        show_user: true
+        default: ASA
```

Comment:
This is funny :) We have two variables with different spelling (_prefix vs _PREFIX). Is it a wrong configuration definition in Beats and can be somehow compacted?

Also, for a normal user this variable may sound a bit odd. You can add a "description" property and describe what is it all about.

Reply:
@adriansr Can you help to explain the reason of the two variables? thanks